rules/bitstreams.bzl

```diff
@@ -0,0 +1,64 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+def _bitstreams_repo_impl(ctx):
+    ctx.execute([
+            "python3",
```

Comment:
We have python rules in the workspace, any reason we can't use a py_binary executable instead of invoking python directly?

Reply:
I [don't think so](https://docs.bazel.build/versions/main/skylark/lib/globals.html#repository_rule(implementation,%20attrs,%20local,%20environ,%20configure,%20remotable,%20doc)):

```
... a repository rule cannot depend on a generated artifact ...
```

Reply:
ahhh got it. thanks for clarifying.

Reply:
Can we file a bug for this? Not using Bazel's python3 breaks hermeticity. We should look for a long-term fix; I'd like to eventually get rid of both `requirements` files.

Alternatively we could rewrite the tool in go and vendor it in as a binary; go tools have the benefit of having system zero dependencies.

Reply:
I was able to use the bazel managed python interpreter at least here: https://github.com/lowRISC/opentitan/blob/d500091e057128ed0121d6f9e2ca95618ff7d745/third_party/python/pip.bzl#L24

Reply:
Done.